src/System.Binary.Base64/System/Binary/Base64Encoder.cs

```diff
@@ -192,10 +198,15 @@ public static bool EncodeInPlace(Span<byte> buffer, int sourceLength, out int by
             return false;
         }
 
-        sealed class ToBase64 : BufferEncoder
+        sealed class ToBase64Utf8 : BufferEncoder
         {
             public override OperationStatus Encode(ReadOnlySpan<byte> source, Span<byte> destination, out int bytesConsumed, out int bytesWritten)
                 => Base64.Encode(source, destination, out bytesConsumed, out bytesWritten);
+
+            public override OperationStatus EncodeInPlace(Span<byte> buffer, int inputLength, out int written)
+                => Base64.EncodeInPlace(buffer, inputLength, out written)?OperationStatus.Done:OperationStatus.DestinationTooSmall;
```

Comment:
Why not change EncodeInPlace to return OperationStatus instead of bool just like DecodeInPlace?

Reply:
EncodeInPlace will go away. It's obsolete. I don't want to change it because I don't want to break Signal-R.